### PR TITLE
Polishing ReadMe file.

### PR DESCRIPTION
**Pull request by elijahtsai**
1. Change the banner file to gif.
2. Redesign the magic trick image.
3. Fixed the mess I made.😂

To: @duraki 
About the description of the magic trick image, Do you have a better sentence or this will be good, your choice!

### CODE REVIEW

**Comment by elijahtsai** [comment]
I can't wait this readme file to publish!!!😂

File: README.md [hunk 1]
```diff
@@ -21,9 +25,12 @@ Sketch.App Patch Tool, brought to you by [@duraki](https://github.com/duraki) &
 ## Magic Trick ✨
 For the people who would like to try the new version, we got you covered. You can pass `-m` argument for the ultimate life-saving trick, which will automagically download latest Sketch app from the official website and patch the bundle, ready to be launched from the Applications folder.
 
-And do it in one command: `bash -c "$(curl -s https://raw.githubusercontent.com/duraki/SketchCrapp/master/sketchcrapp.sh -o -)" -O -m`
+[@gordio](https://github.com/gordio) added<br />And do it in one command: `bash -c "$(curl -s https://raw.githubusercontent.com/duraki/SketchCrapp/master/sketchcrapp.sh -o -)" -O -m`
```
Comment (duraki) [inline README.md:28]: I'd remove `[@gordio](https://github.com/gordio) added<br />`. He is already listed as a contributor on the README commits.

Screenshot 2021-02-09 at 09 47 37

Comment (duraki) [inline README.md:28]: Rewrite as:
---


One-liner script to install latest Sketch version and automatically patch it:

```
bash -c "$(curl -s https://raw.githubusercontent.com/duraki/SketchCrapp/master/sketchcrapp.sh -o -)" -O -m
```

**Review by duraki — CHANGES_REQUESTED** [review]
The gif is really cool haha! Very interactive in my opinion, and fits very well. Please fix the things I mentioned in the review and we can merge it to master ASAP.

File: README.md (hunk 1 above)
Comment (elijahtsai) [reply README.md:28]: Ok

**Pull request merged**